utils/config/daos_server.yml

```diff
@@ -290,7 +290,7 @@
 #  bdev_list: ["0000:5d:05.5]
 
 #-
-#  # Rank to be assigned as identifier for server.
+#  # Rank to be assigned as identifier for this engine.
```

Comment:
@johannlombardi @mjmac is it time to change `rank` to `instance` or something similar now, given that this isn't actually the unique rank?

Reply:
This is in fact a rank in the system sense. Normally the ranks are assigned by the MS, but we do support allowing the rank to be set in the config.